transform-plugins/src/main/java/co/cask/hydrator/plugin/CSVParser.java

```diff
@@ -282,11 +284,19 @@ public Config(@Nullable String format, @Nullable Character delimiter, String fie
 
     private void validate() {
 
+      if (format == null) {
+        if (delimiter != null && delimiter != 0) {
+          throw new IllegalArgumentException("Custom delimiter can only be used for format option 'Custom'.");
+        }
+        return;
+      }
+
       // Check if format is one of the allowed types.
       if (!format.equalsIgnoreCase("DEFAULT") && !format.equalsIgnoreCase("EXCEL") &&
         !format.equalsIgnoreCase("MYSQL") && !format.equalsIgnoreCase("RFC4180") &&
         !format.equalsIgnoreCase("Tab Delimited") && !format.equalsIgnoreCase("Pipe Delimited")
-        && !format.equalsIgnoreCase("Custom")) {
+        && !format.equalsIgnoreCase("Custom") && !format.equalsIgnoreCase("PDL") &&
```

Comment:
style nit: should not the leading `&&` be placed at the end of the previous line?